pipeline/api-service/overlays/multi-user/config-map.yaml

```diff
@@ -19,8 +19,10 @@ data:
       },
       "InitConnectionTimeout": "6m",
       "DefaultPipelineRunnerServiceAccount": "default-editor",
-      "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_HOST": "ml-pipeline-ml-pipeline-visualizationserver",
-      "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_PORT": 8888,
+      "VisualizationService":{
```

Comment:
I wrote an example that can avoid this duplication: https://github.com/Bobgy/kustomize-configmap-merge-example
Maybe not right now, but let's switch to that pattern soon.

Reply:
Thanks for example. Looks good!
Sound like it should be a separate effect to migrate the whole config, so leave it later.

Reply:
Agree